skimage/data/_binary_blobs.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+from ..filters import gaussian_filter
+
+
+def binary_blobs(length=512, blob_size_fraction=0.1, n_dim=2,
+                 volume_fraction=0.5, seed=None):
+    """
+    Generate synthetic binary image with several blob-like rounded objects.
```

Comment:
I'm not a native speaker, but I hope someone who is will correct us if we're on the wrong path :)